karate-tests/src/test/java/patients/TestSmoke.java

```diff
@@ -0,0 +1,23 @@
+package patients;
+
+import com.intuit.karate.Results;
+import com.intuit.karate.Runner;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+
+public class TestSmoke {
+    
+    @Test
+    void testSmoke() {
+        Results results = Runner.path("classpath:patients")
+            .karateEnv("veit07")
```

Comment:
Yes this is incorrect. Smoke runs against int. I'll add a new config for this.

Reply:
That reminds me, we used to run tests in ref but removed the pipeline stage as data got inconsistent between the environments. Given your approach on checking the response against the schema, it might be worth revisiting. Maybe one to pick up with Claudiu.